### PR TITLE
Add documentation of log levels

### PR DESCRIPTION
**Pull request by tfoote**
This is a follow up to the [Aggregate Foxy Message API Review](https://github.com/ros2/common_interfaces/pull/86)

Specifically: https://github.com/ros2/common_interfaces/pull/86/files#r393330197

### CODE REVIEW

File: rcl_interfaces/msg/Log.msg [hunk 1]
```diff
@@ -1,5 +1,13 @@
 ##
 ## Severity level constants
+## 
+## These logging levels follow the Python Standard
+## https://docs.python.org/3/library/logging.html#logging-levels
+## And are implemented in rcutils as well
+## https://github.com/ros2/rcutils/blob/35f29850064e0c33a4063cbc947ebbfeada11dba/include/rcutils/logging.h#L164-L172
+## This leaves space for other logging levels to be inserted in the middle in the future.
```
Comment (dirk-thomas) [inline rcl_interfaces/msg/Log.msg]: Not only in the future but custom user defined levels.

File: rcl_interfaces/msg/Log.msg [hunk 2]
```diff
@@ -1,5 +1,13 @@
...
+## Since there are several other logging enumeration standard for different implementations.
+## Other logging implementations may need to provide level translations to match their internal implementations.
```
Comment (dirk-thomas) [inline rcl_interfaces/msg/Log.msg]: Maybe use the term "mapping" instead of "translation" to be more specific?

File: rcl_interfaces/msg/Log.msg [hunk 3]
```diff
@@ -1,5 +1,13 @@
 ##
 ## Severity level constants
+## 
+## These logging levels follow the Python Standard
+## https://docs.python.org/3/library/logging.html#logging-levels
+## And are implemented in rcutils as well
+## https://github.com/ros2/rcutils/blob/35f29850064e0c33a4063cbc947ebbfeada11dba/include/rcutils/logging.h#L164-L172
+## This leaves space for other logging levels to be inserted in the middle in the future.
+## Since there are several other logging enumeration standard for different implementations.
```
Comment (dirk-thomas) [inline rcl_interfaces/msg/Log.msg]: This doesn't seem to be a complete sentence?

**Comment by hidmic** [comment]
@tfoote friendly ping.

**Review by dirk-thomas — APPROVED** [review]

File: rcl_interfaces/msg/Log.msg [hunk 4]
```diff
@@ -1,5 +1,14 @@
...
 ## Severity level constants
+## 
+## These logging levels follow the Python Standard
+## https://docs.python.org/3/library/logging.html#logging-levels
+## And are implemented in rcutils as well
+## https://github.com/ros2/rcutils/blob/35f29850064e0c33a4063cbc947ebbfeada11dba/include/rcutils/logging.h#L164-L172
+## This leaves space for other standard logging levels to be inserted in the middle in the future,
+## as well as custom user defined levels.
+## Since there are several other logging enumeration standard for different implementations,
+## other logging implementations may need to provide level mappings to match their internal implementations.
```
Comment (dirk-thomas) [inline rcl_interfaces/msg/Log.msg:11]: Out of curiosity: why two `#`? Are these comments still being extracted properly and used in the generated code without any leading `#`?

Comment (tfoote) [reply rcl_interfaces/msg/Log.msg:11]: I was just copying the existing style.

**Comment by dirk-thomas** [comment]
Please double check the docblocks in the generated code. That was also part of the review if I recall correctly.

**Pull request merged**